.circleci/config.yml

```diff
@@ -269,6 +269,41 @@ jobs:
       - coverage:
           flags: 'test_<< parameters.old_pg_major >>_<< parameters.new_pg_major >>,upgrade'
 
+  test-pytest:
+    description: Runs pytest based tests
+    parameters:
+      pg_major:
+        description: 'postgres major version'
+        type: integer
+      image:
+        description: 'docker image to use as for the tests'
+        type: string
+        default: citus/failtester
+      image_tag:
+        description: 'docker image tag to use'
+        type: string
+    docker:
+      - image: '<< parameters.image >>:<< parameters.image_tag >><< pipeline.parameters.image_suffix >>'
+    working_directory: /home/circleci/project
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - install_extension:
+          pg_major: << parameters.pg_major >>
+      - configure
+      - enable_core
+      - run:
+          name: 'Run pytest'
+          command: |
+            gosu circleci \
+              make -C src/test/regress check-pytest
+          no_output_timeout: 2m
+      - stack_trace
+      - coverage:
+          flags: 'test_<< parameters.pg_major >>'
```

Comment:
Looking at all the other test jobs each of them updates the relevant pgmajor flag for the version that its testing, in addition to adding to the flag for the type of test itself (e.g. `upgrade`). This makes sense, so we can see if we're missing coverage for certain versions (not that I think anyone ever looks at this though). So I kept it at is, but also added a `pytest` flag, so that we can see how much coverage our pytest tests generate.